package.json

```diff
@@ -178,13 +178,14 @@
     "defaults"
   ],
   "scripts": {
-    "build": "concurrently npm:gen:op npm:gen:shadowdom",
+    "build": "\"concurrently npm:gen:op npm:gen:shadowdom\" && npm run gen:types",
```

Comment:
i'm trying to make `tsc` run after concurrently is finished. it works fine locally (on windows) but keeps failing in CI 😕 maybe i'm missing something obvious